bff-tiltaksarrangor/src/main/kotlin/no/nav/amt/tiltak/bff/tiltaksarrangor/VeilederController.kt

```diff
@@ -100,6 +105,20 @@ class VeilederController (
 		return tilgjengelige.map { TilgjengeligVeilederDto(it.id, it.fornavn, it.mellomnavn, it.etternavn) }
 	}
 
+	@ProtectedWithClaims(issuer = Issuer.TOKEN_X)
+	@GetMapping("/veileder/deltakerliste")
+	fun hentDeltakerliste(): List<VeiledersDeltakerDto> {
+		val ansatt = hentInnloggetAnsatt()
+		arrangorAnsattTilgangService.shouldHaveRolle(ansatt.personligIdent, ArrangorAnsattRolle.VEILEDER)
+
+		val gjennomforingIder = arrangorAnsattTilgangService.hentGjennomforingIder(ansatt.personligIdent)
```

Comment:
Du må ha tilgang til gjennomføringen i seg selv for å få opp deltakere? skjønner ikke det helt

Reply:
Må trykke på legg til på gjennomføringen? Var ikke det vi diskuterte også endte med at det er tungvint?

Reply:
ah, jeg trodde det var et krav at veileder også må ha tilgang til selve gjennomføringen. Hvis det holder at noen har satt veilederen som veileder for en gitt deltaker så er det gull, da kan jeg fjerne denne :D